### PR TITLE
ott 0.25 (new formula)

### PR DESCRIPTION
**Pull request by tchajed**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This doesn't technically pas `brew audit --strict --new-formula ott` because there aren't quite enough stars/followers on the GitHub repo.  Should I wrangle up some more stars for this PL tool before adding it to Homebrew?

### CODE REVIEW

File: Formula/ott.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  depends_on "ocaml" => :build
+
+  def install
+    system "make", "world"
+    bin.install "bin/ott"
+    pkgshare.install "examples"
+
+    site_lisp=pkgshare/"emacs/site-lisp/ott"
+    mkdir_p site_lisp
```
Comment (bfontaine) [inline Formula/ott.rb]: You can remove this line. `site_lisp.install` will automatically create the directory if it doesn’t already exist.

File: Formula/ott.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  homepage "https://www.cl.cam.ac.uk/~pes20/ott/"
+  url "https://github.com/ott-lang/ott/archive/0.25.tar.gz"
+  sha256 "c6abbbeb8cd44dc973d45d30bc5a7e42e212f2feba45c8e0489fab3c3cbf0d78"
+  head "https://github.com/ott-lang/ott.git"
+
+  depends_on "ocaml" => :build
+
+  def install
+    system "make", "world"
+    bin.install "bin/ott"
```
Comment (bfontaine) [inline Formula/ott.rb]: Is there no `make install` we could use?

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/ott.rb (hunk 1 above)
Comment (tchajed) [reply Formula/ott.rb]: Fixed, thanks!

File: Formula/ott.rb (hunk 2 above)
Comment (tchajed) [reply Formula/ott.rb]: Not really; this already builds the `install` target, which is hard-coded to copy the binary to `bin/ott`.

**Comment by fxcoudert** [comment]
@brewtestbot test this please

**Pull request closed without merging**